gsoc/models.py

```diff
@@ -0,0 +1,204 @@
+from django.db import models
+from eventtools.models import BaseEvent, BaseOccurrence
+
+
+class Gsoc(models.Model):
+    count = models.IntegerField(primary_key=True)
+
+    def __str__(self):
+        return str(self.count)
+
+
+class GsocEvent(BaseEvent):
+    title = models.TextField()
+
+    def __str__(self):
+        return self.title
+
+
+class GsocOccurrence(BaseOccurrence):
+    event = models.ForeignKey(GsocEvent)
+
+    def __str__(self):
+        return self.event
+
+
+class Program(models.Model):
+    year = models.IntegerField(primary_key=True)
+    state = models.CharField(max_length=100)
+    is_current = models.BooleanField()
+    org_signup_open = models.BooleanField()
```

Comment:
you have a second `org_signup_open` below ;-)

Many of these bools are like that; I guess they are derived ; i.e. this bool is enabled if the current date is in the date range for the event below

Reply:
> you have a second org_signup_open below ;-)

Then, what name should I give the second one?

Reply:
the bool doesnt need to exist; it can be derived.